node/test/runtime/Cargo.toml

```
@@ -0,0 +1,46 @@
+[package]
+name = "polkadot-runtime-test"
+version = "0.8.28"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+test-runner = { git = "https://github.com/paritytech/substrate", branch = "substrate-test-runner" }
```

please make sure to run `cargo check` to update the `Cargo.lock` so that bot will be able to merge this properly
CI here will be red anyway, but the build should be checked on the substrate side 
```suggestion
test-runner = { git = "https://github.com/paritytech/substrate", branch = "master" }
```